tests/django_hstore_tests/tests/test_schema_mode.py

```diff
@@ -65,6 +66,17 @@ def test_dict_get(self):
         self.assertEqual(d.get('default_test', 'default'), 'default')
         self.assertIsNone(d.get('default_test'))
 
+    def test_save_and_reload_custom_type(self):
+        d = SchemaDataBag()
+        v = CustomType('some value')
+        d.custom = v
+        self.assertEqual(d.custom, v)
+
+    def test_serializing_custom_type(self):
+        d = SchemaDataBag()
+        d.custom = CustomType('some value')
+        self.assertEqual(dict.get(d.data, 'custom'), '[some value]')
```

Comment:
I got the previous test to pass but here I have some doubts.

Isn't `dict.get(d.data, 'custom')` the same as `d.data.get('custom')`?
If that is correct it should return the `CustomType` instance, not the serialized value "[some value]".
If I'm not wrong the serialized value must be used only before storing the value into the database, which would mean this is not the right way to test for the desired behaviour.
What do you think?